### DISCUSSION TITLE
React Navigation without React Native

### DISCUSSION THREAD

**Discussion by jaskp** [react-navigation/react-navigation#12138] [Q&A]
Hi, v7 is looking great, I was wondering what's the team's stance on ever supporting React Navigation in pure React apps without React Native? Thanks!

**Comment by JohnImril** [comment]
Hello, why do you need react-navigation for pure react applications, there is react-router-dom for them?

**Reply by jaskp** [reply]
Because react router nor tanstack router nor wouter don't have quite the same API as react navigation

**Comment by satya164** [comment]
`@react-navigation/core` is platform agnostic. It runs anywhere React runs. You can build custom navigators using it for the platform you want. For example, there's one for NativeScript https://react-nativescript.netlify.app/docs/core-concepts/navigation
